IPython/core/magic.py

```diff
@@ -675,6 +676,9 @@ def __call__(self, *args, **kwargs):
                              "magic aliases cannot call themselves.")
         self._in_call = True
         try:
-            return fn(*args, **kwargs)
+            if self.magic_params:
```

Comment:
It looks like an alias created with parameters can't be called with any more parameters - you either pass parameters in while creating the alias,  or when you call it, but not both. That may well be a reasonable restriction, but I would have expected that any parameters specified when you call it would be added after the parameters defined with the alias.